fixtures/webstudio-remix-vercel/app/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const assetBaseUrl = "/assets/";
+export const imageBaseUrl = "/assets/";
```

Comment:
Should we leave a comment, that this should match with this step in prebuild.
https://github.com/webstudio-is/webstudio/blob/main/packages/cli/src/config.ts#L14

Reply:
<img width="755" alt="image" src="https://github.com/webstudio-is/webstudio/assets/5077042/41a7aa86-241f-469c-b726-64795a808ce7">

Im gonna remove that step in next iterations.

Reply:
cool 👍

Reply:
As of now found the only way is to rename constants.ts to constants.mjs and eval using node 
```
node --experimental-specifier-resolution=node --input-type=module --eval "import * as consts from './app/constants.mjs'; console.log(JSON.stringify(consts))"
```